geode-core/src/main/java/org/apache/geode/internal/VersionDescription.java

```diff
@@ -119,7 +119,7 @@ public String getProperty(String key) {
   }
 
   private String getNativeCodeVersion() {
-    return SmHelper.getNativeVersion();
+    return "native code unavailable";
```

Comment:
Instead of always logging in the banner that "native code" is unavailable, I think it is time to remove this message. Since we don't have native code this message is misleading. It can make someone think they have configured things wrong. I think you should go to the one caller of this method and delete that message and then get rid of getNativeCodeVersion